src/core/ext/transport/chaotic_good/server/chaotic_good_server.h

```diff
@@ -126,14 +126,20 @@ class ChaoticGoodServerListener final : public Server::ListenerInterface {
     int32_t data_alignment_;
   };
 
-  void Start(Server*, const std::vector<grpc_pollset*>*) override {
-    StartListening().IgnoreError();
-  };
+  void Start() override { StartListening().IgnoreError(); };
 
   channelz::ListenSocketNode* channelz_listen_socket_node() const override {
     return nullptr;
   }
 
+  void SetServerListenerState(RefCountedPtr<Server::ListenerState>) override {}
+
+  const grpc_resolved_address* resolved_address() const override {
+    // chaotic good doesn't use the new ListenerState interface yet.
+    Crash("Unimplemented");
```

Comment:
We would need to implement the LogicalConnection interface (SendGoAway and DisconnectImmediately). This would then allow us to use the ListenerState object to track logical connections by adding/removing them.